contracts/LosslessGovernance.sol

```diff
@@ -27,6 +27,8 @@ contract LosslessGovernance is ILssGovernance, Initializable, AccessControlUpgra
 
     uint256 public compensationPercentage;
 
+    uint256 public revsharePercent;
```

Comment:
Contract is upgradable, this would break stuff on the deployed testnets.

Reply:
Addressed on new commits